cmd/internal/codegen/parse/crd.go

```diff
@@ -173,6 +173,24 @@ var primitiveTemplate = template.Must(template.New("map-template").Parse(
     {{ if .Format -}}
     Format: "{{ .Format }}",
     {{ end -}}
+    {{ if .Enum -}}
+    Enum: {{ .Enum }},
+    {{ end -}}
+    {{ if .MaxLength -}}
+    MaxLength: {{ .MaxLength }},
```

Comment:
I took a quick look and noticed something. `MaxLength` is a `*int64` type. You need some function here to convert the value to the correct type.

Reply:
Good catch~ Thank you.

Reply:
In my e2e test, I found an issue in crd generation and evaluation, I am going to fix it in this PR.